docs/configuration.md

```diff
@@ -0,0 +1,109 @@
+| `LIMIT_DURATION`                  | "60000"       | The maximum duration in ms applied to IP-method based rate limits.   |
+| `MIRROR_NODE_LIMIT_PARAM`         | "100"         | The mirror node custom limit value to be set on GET requests. This optimizes the flow to reduce the number of calls made to the mirror node by setting a limit larger than it's default limit.   |
+| `MIRROR_NODE_RETRIES`             | "3"           | The maximum number of retries on a GET request to the mirror node when an acceptable error code is returned.   |
+| `MIRROR_NODE_RETRY_DELAY`         | "250"         | The dealy in ms between retry requests.   |
+| `MIRROR_NODE_URL`                 | ""            | The Mirror Node API endpoint. Official endpoints are Previewnet (https://previewnet.mirrornode.hedera.com/v1/api), Testnet (https://testnet.mirrornode.hedera.com/v1/api), Mainnet (https://mainnet-public.mirrornode.hedera.com/v1/api). See [Mirror Node REST API](https://docs.hedera.com/hedera/sdks-and-apis/rest-api)   |
+| `SDK_REQUEST_TIMEOUT`             | "10000"       | The complete timeout for running the SDK `execute()` method. This controls the GRPC channel timeout cofig when querying with network nodes.   |
+| `TIER_1_RATE_LIMIT`               | "100"         | Maximum restrictive request count limit used for expensive endpoints rate limiting.   |
+| `TIER_2_RATE_LIMIT`               | "800"         | Maximum moderate request count limit used for non expensive endpoints.   |
+| `TIER_3_RATE_LIMIT`               | "1600"        | Maximum relaxed request count limit used for static return endpoints.   |
+
+
+## Testing
+
+The following table lists the available properties along with their default values for the tests utilized in the [Server](/packages/server/) and [Relay](/packages/relay/) packages. 
+Unless you need to set a non-default value, it is recommended to only populate overridden properties in the custom `.env`.
+
+| Name                          | Default       | Description                                                   |
+|-------------------------------|---------------|---------------------------------------------------------|
+| `LOCAL_NODE`      | ""        | Flag if relay is hosted in the Hedera local node setup.   |
+| `E2E_RELAY_HOST`  | ""        | Remote relay url to point to.   |
+| `DEV_MODE`        | "false"   | Flag if relay should operate in developer optimization mode.   |
+
+For test context additional fields need to be set. The following example showcases a `hedera-local-node` instance (where values match those noted on [Local Node Network Variables](https://github.com/hashgraph/hedera-local-node#network-variables)
+
+```.env
+HEDERA_NETWORK={"127.0.0.1:50211":"0.0.3"}
+OPERATOR_ID_MAIN=0.0.2
+OPERATOR_KEY_MAIN=302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137
```

Comment:
These are the values for local node.  Would it be helpful to highlight that for previewnet/testnet you can get these values from the [portal](http://portal.hedera.com/login)?

Reply:
Will add

Reply:
Done